include/ros_phoenix/base_component.hpp

```diff
@@ -212,7 +212,7 @@ namespace ros_phoenix
 
         void onTimer()
         {
-            ctre::phoenix::unmanaged::FeedEnable(this->watchdog_ms_);
+            ctre::phoenix::unmanaged::Unmanaged::FeedEnable(this->watchdog_ms_);
```

Comment:
Can we get a link to the changelog here with the version that this changed? Will be useful for reference